Dockerfile

```diff
@@ -14,12 +14,13 @@ ENTRYPOINT [ "/sbin/tini", "--" ]
 FROM base AS build-base
 WORKDIR /build
 COPY package.json package-lock.json prisma ./
-RUN npm ci --omit=dev --frozen-lockfile \
- && npx prisma generate
+RUN npm ci --include=dev --frozen-lockfile 
+RUN npx prisma generate
 
 FROM build-base AS build
 COPY --chown=node:node . ./
 RUN npm run build
+RUN npm prune --production
 
 FROM build-base AS dev-build
```

Comment:
If we are going to build it like this with ALL dependencies already in `build-base` stage and prune it for prod, there is no need for this stage. We can drop it altogether and in `dev` stage for [COPY](https://github.com/bratislava/nest-prisma-template/pull/105/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R30) just do

```Dockerfile
COPY --chown=node:node --from=build-base /build/node_modules ./node_modules
```

Or, if we all agree that `dev` stage is **only** used when you mount your directory, we can drop entire `COPY` [directive line](https://github.com/bratislava/nest-prisma-template/pull/105/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R30).

Reply:
It might be more efficient if you add a commit with these changes as I don't have the necessary context for this change